charts/pmm/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: pmm
+description: A Helm chart for Percona Monitoring and Management (PMM)
+type: application
+version: 0.1.0
+appVersion: "2.27.0"
+home: https://github.com/percona/pmm
+maintainers:
+  - name: cap1984
+    email: ivan.pylypenko@percona.com
```

Comment:
@cap1984 @tplavcic I added you folks just as example of other charts :) I wonder who else would be maintaining this, or just PMM team.
CC @BupycHuk